js/plates-view/render-cross-section.js

```diff
@@ -69,6 +75,24 @@ function drawMarker (ctx, crustPos) {
   ctx.fill()
 }
 
+function drawEarthquake (ctx, xPos, elevation, earthquake) {
+  const earthquakeSize = (1 + Math.ceil(earthquake.magnitude))
+  const strokeWidth = earthquakeSize * 0.1
+  const x = scaleX(xPos)
+
+  // Not ideal depth calculation until actual depth is in the correct range
+  const adjustedDepth = elevation - earthquake.depth
```

Comment:
Why isn't it "ideal" and "adjusted"? I think it's all correct here and shouldn't require any more changes. Now, it's only model tweaking.
I'd say it's just "earthquake elevation" (by elevation I mean anything between 1 and -3.3, as that's the elevation range in our model).